strongbox-web-core/src/main/java/org/carlspring/strongbox/web/HeaderMappingFilter.java

```diff
@@ -55,14 +62,27 @@
     private static final String USER_AGENT_MAVEN = "Maven";
 
     private Map<String, String> userAgentMap = new HashMap<>();
+    private Map<String, String> layoutMap = new HashMap<>();
+
+    @Inject
+    private ConfigurationManager configurationManager;
 
     @Override
     public void init(FilterConfig filterConfig)
         throws ServletException
     {
+        //Do nothing
+    }
+
+    @PostConstruct
+    public void postConstruct()
```

Comment:
added TODO about this, I think this is not critical for now, and we can think how to implement such auto configuration. Maybe we don't need a `user-agent` header at all and should use some kind of custom header for this like `stromgbox-layout`